apps/website/src/pages/docs/alert.mdx

```diff
@@ -17,65 +17,78 @@ import * as AllExamples from 'examples';
   <AllExamples.AlertMainExample client:load />
 </LiveExample>
 
-An alert is an element that notifies the user of something important that is not user initiated. If you need to alert the user because of something they did, consider using a toast notification or an inline message. Instead, an alert is used to notify the user of something unrelated to what they just did. Examples of this might be announcing a new feature within the product, or alerting the user of scheduled server maintenance and downtime.
+An alert is an element that notifies the user of something important that is not user initiated. If you need to alert the user because of something they did, consider using a [toast](toast) notification or an inline message. Instead, an alert is used to notify the user of something unrelated to what they just did. Examples of this might be announcing a new feature within the product, or alerting the user of scheduled server maintenance and downtime.
 
-## Appearance
+## Usage
 
-The alert message must be concise, we recommend no more than 256 characters. If you feel the need to explain in further detail, you may use an optional hyperlink at the end of the alert message to explain more on a new page. You may also include an optional close icon (close icon is required for sticky alerts).
+The alert message must be concise, we recommend no more than 256 characters. If you feel the need to explain in further detail, you may use an optional [hyperlink](#alertaction) at the end of the alert message to explain more on a new page. You may also include an optional close icon (close icon is required for [sticky](#sticky) alerts).
```

Comment:
I liked this line "If you need to alert the user because of something they did, consider using a [toast](toast) notification or an inline message." and felt it was important. Can we add it to the very end of the introduction?